assets/js/modules/analytics/datastore/accounts.test.js

```diff
@@ -48,6 +55,15 @@ describe( 'modules/analytics accounts', () => {
 		store = registry.stores[ STORE_NAME ].store;
 
 		apiFetchSpy = jest.spyOn( { apiFetch }, 'apiFetch' );
+
+		/* eslint-disable no-restricted-globals */
+		windowLocationSpy = jest.spyOn( window.location, 'assign' );
```

Comment:
Maybe `locationAssignSpy` would be a more precise name? `windowLocationSpy` sounds like it would cover all functions of `window.location`.

Reply:
Good idea, I think this can be changed to just `location` as well, I'll give that a try.

Reply:
> Good idea, I think this can be changed to just `location` as well, I'll give that a try.

Agreed. I think `locationSpy` would be better here. One thing to keep in mind (although I don't think it makes a difference here) is that Jest proxies the calls through to the real implementation, so if you don't want that you can set your own implementation.